x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/rum/field_formats.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FieldFormat } from '../../types';
+import {
+  FCP_FIELD,
+  FID_FIELD,
+  LCP_FIELD,
+  TBT_FIELD,
+  TRANSACTION_DURATION,
+} from '../constants/elasticsearch_fieldnames';
+
+export const rumFieldFormats: FieldFormat[] = [
+  {
+    field: TRANSACTION_DURATION,
+    format: {
+      id: 'duration',
+      params: {
+        inputFormat: 'microseconds',
+        outputFormat: 'asSeconds',
+        showSuffix: true,
```

Comment:
It's technically @elastic/kibana-app-services, but I am the person who introduced this `showSuffix` flag. It is very difficult to change the existing formatters because we consider any change to the output a breaking change, so we can add new options if they don't break old ones.

Reply:
@wylieconlon i have another question, do you think it make sense to introduce a short suffix form? like instead of a full work 'Milliseconds' we want just 'ms' and same goes for 'Seconds'
would that be a breaking change? and how difficult it would be to add that on top of what we already have.